service/src/main/java/com/fecklessweasel/service/PopulateServlet.java

```diff
@@ -0,0 +1,139 @@
+package com.fecklessweasel.service;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.RequestDispatcher;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import com.fecklessweasel.service.datatier.SQLSource;
+import com.fecklessweasel.service.datatier.SQLInteractionInterface;
+import com.fecklessweasel.service.objectmodel.ServiceException;
+import com.fecklessweasel.service.objectmodel.ServiceStatus;
+import com.fecklessweasel.service.objectmodel.User;
+import com.fecklessweasel.service.objectmodel.UserSession;
+import com.fecklessweasel.service.objectmodel.Course;
+import com.fecklessweasel.service.objectmodel.Department;
+import com.fecklessweasel.service.objectmodel.University;
+import com.fecklessweasel.service.objectmodel.FileMetadata;
+
+/**
+ * A servlet that will fill the Database with testing data when posted to
+ * new users.
+ * @author Hayden Schmackpfeffer
+ */
+@WebServlet("/servlet/populate")
+public final class PopulateServlet extends HttpServlet {
```

Comment:
This works, but we really should have some way to block it or turn it off at release deployment. We don't want people to be able to at will populate the DB with stuff.